### ISSUE TITLE
Show empty array [] instead of nothing

### ISSUE THREAD

**Issue by ilanbiala** [jeromelebel/MongoHub-Mac#189]
Right now an empty array is just blank, it would be nice if it showed `[]` instead.

**Comment by ilanbiala** [comment]
@jeromelebel what do you think about this? Can we get a quick fix in?

**Comment by jeromelebel** [comment]
yes, that can be fixed. But it would be nice to have something done correctly. I mean it would be better to have "[ object1, object2..."  when the array is not empty.
But unfortunately, I think the aggregation is more important to fix.
